string cmd
string markerName
string newMarkerName
---
string message